src/machine/machine-dbus.c

```diff
@@ -928,6 +930,22 @@ int bus_machine_method_copy(sd_bus_message *message, void *userdata, sd_bus_erro
         if (r < 0)
                 return r;
 
+        call_contains_flags = sd_bus_message_is_method_call(message, NULL, "CopyFromMachineWithFlags") || sd_bus_message_is_method_call(message, NULL, "CopyToMachineWithFlags")
+                || sd_bus_message_is_method_call(message, NULL, "CopyFromWithFlags") || sd_bus_message_is_method_call(message, NULL, "CopyToWithFlags");
```

Comment:
My bad, it turns out there is a way to get the member from the D-Bus message (`sd_bus_message_get_member()`). You can use that and then check if that string ends with "WithFlags". We have the `endswith()` function for that.